src/firmware_new/boards/frankie_v1/io/gpio_pin.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "stm32h7xx_hal.h"
+
+typedef struct GpioPin GpioPin_t;
+
+/**
+ * A GPIO pin being "Active High" or "Active Low" simply determines whether high or low
+ * voltage corresponds to 1. So for example, "Active High" means that a high voltage
+ * corresponds to 1, a low voltage corresponds to 0.
+ */
+typedef enum
+{
+    ACTIVE_HIGH,
+    ACTIVE_LOW
+} GpioPinActiveState;
```

Comment:
Hmm, what would you add? It's a bit hard for me to explain when I already understand it, but definitely something I wanted to make clear.

Reply:
I think just adding to the comment above would be enough for those that might get confused. It figured it out pretty quick, I just think it would be good to confirm in a comment.

Reply:
Sorry, could you post exactly what the final overall comment would be? It's not clear to me from the above what I should add.

Reply:
On second thought I think this is fine, I can't think of anything to write that adds _that_ much benefit without potentially adding confusion.